request-unix.go

```diff
@@ -22,6 +24,14 @@ func testOsSys(sys interface{}) error {
 	return nil
 }
 
-func toLocalPath(p string) string {
+func toLocalPath(workDir, p string) string {
+	if workDir != "" {
+		if !filepath.IsAbs(p) && !path.IsAbs(p) {
```

Comment:
`filepath.Abs` is equivalent to `path.IsAbs` under build tag `unix`.

Reply:
I wanted to keep the code consistent across platforms, but I'll simplify 👍🏻

Reply:
Platform-specific code should always be code specific to the platform.

If the code could be written generically, then it wouldn’t need to be put in platform-specific code.